front/lib/api/assistant/conversation.ts

```diff
@@ -309,11 +310,13 @@ async function createOrUpdateParticipation({
       },
     });
     if (participant) {
-      return participant.update({
+      participant.changed("updatedAt", true);
```

Comment:
Do we need that line?

Reply:
Yes it's not updating without it, some dark sequelize magic 😕